packages/test/test-end-to-end-tests/src/test/cellEndToEndTests.spec.ts

```diff
@@ -279,6 +280,43 @@ describeNoCompat("SharedCell orderSequentially", (getTestObjectProvider) => {
         containerRuntime = dataObject.context.containerRuntime as ContainerRuntime;
     });
 
+    it("Should rollback set", async () => {
+        let error: Error | undefined;
+        try {
+            containerRuntime.orderSequentially(() => {
+                sharedCell.set(0);
+                throw new Error(errorMessage);
+            });
+        } catch (err) {
+            error = err as Error;
+        }
+
+        assert.notEqual(error, undefined, "No error");
+        assert.equal(error?.message, errorMessage, "Unexpected error message");
+        assert.equal(containerRuntime.disposed, false);
+        assert.equal(sharedCell.get(), undefined);
+        // rollback
+    });
+
+    it("Should rollback delete", async () => {
+        sharedCell.set("old");
+        let error: Error | undefined;
+        try {
+            containerRuntime.orderSequentially(() => {
+                sharedCell.delete();
+                throw new Error(errorMessage);
+            });
+        } catch (err) {
+            error = err as Error;
+        }
+
+        assert.notEqual(error, undefined, "No error");
+        assert.equal(error?.message, errorMessage, "Unexpected error message");
+        assert.equal(containerRuntime.disposed, false);
+        assert.equal(sharedCell.get(), "old");
+        // rollback
+    });
+
     itExpects("Closes container when rollback fails",
```

Comment:
Now that you've implemented it, this test should fail. Could you move this to another DDS that doesn't yet have rollback?

Reply:
I meant to put a similar test in another DDS that doesn't support rollback to ensure the unsupported case works as expected. Counter would work. Anything except string, map, directory, and cell.